src/Okta.Idx.Sdk/IdxUrlHelper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Okta.Idx.Sdk
+{
+    public static class IdxUrlHelper
+    {
+
+        public static string GetNormalizedUriString(string issuer, string resourceUri)
```

Comment:
This may handle a prior question, but what happens if the issuer does not have the trailing `/` in the config?

Reply:
It's probably worth writing a unit test to make sure correct assumptions are being made.  I'll add a test.

Reply:
I believe that Path.Combine will ensure no wackiness occurs, but I'll still write a test or two to confirm.